package.json

```diff
@@ -28,7 +34,7 @@
     "fix:pkgs": "manypkg fix",
     "version:pkgs": "changeset version",
     "clean:emotion-types": "rm -rf node_modules/@emotion/core/types",
-    "postinstall": "yarn clean:emotion-types && yarn check:pkgs",
+    "postinstall": "yarn clean:emotion-types && yarn check:pkgs && preconstruct dev",
```

Comment:
`preconstruct dev` is the piece that connects everything together for local dev. For each package, it creates `dist` output that imports from the package's `src` entrypoint. No need to build anything locally! 

And since `postinstall` is connected to `yarn install`, it will automatically do this when the user installs packages. It constantly keeps the references up to date, and also will automatically be run when a user clones the repo and installs for the first time.

Big improvement in the contribution process imo

Reply:
Maybe worth moving the `preconstruct dev` before `yarn check:pkgs`? If the check fails you will at least have those preconstruct links already working.